__tests__/routes/applicationRoutes.test.js

```diff
@@ -82,36 +82,36 @@ describe('Application Routes', () => {
     });
   });
 
-  //Test Application Approval
-  describe('PUT /api/application/:idapprove', () => {
-    it('Should approve an application', async () => {
-      const fakeId = newUser._id;  // Assuming newUser is already defined in your test setup
-
-      await request(app)
-        .put(`/api/application/${fakeId}approve`)  // Updated the route
-        .expect(200);  // Expect a successful approval
-
-      const updatedNewContentCreator = await db.collection('content-creators').findOne({ baseUser: fakeId });
-      console.log(updatedNewContentCreator);
-      expect(updatedNewContentCreator.approved).toBe(true);
-    });
-  });
-
-  //Test Application Rejection
-  describe('PUT /api/application/:id/reject', () => {
-    it('Should reject an application with a reason', async () => {
-      const fakeId = newUser._id;  // Assuming newUser is already defined in your test setup
-      const reason = 'Not meeting the criteria';
-
-      await request(app)
-        .put(`/api/application/${fakeId}/reject`)  // Updated the route
-        .send({ reason })  // Sending the reason in the request body
-        .expect(200);  // Expect a successful rejection
-
-      const updatedNewContentCreator = await db.collection('content-creators').findOne({ baseUser: fakeId });
-      expect(updatedNewContentCreator.rejected).toBe(true);
-    });
-  });
+  // //Test Application Approval
+  // describe('PUT /api/application/:id?approve', () => {
+  //   it('Should approve an application', async () => {
+  //     const fakeId = newUser._id;  // Assuming newUser is already defined in your test setup
+
+  //     await request(app)
+  //       .put(`/api/application/${fakeId}approve`)  // Updated the route
+  //       .expect(200);  // Expect a successful approval
+
+  //     const updatedNewContentCreator = await db.collection('content-creators').findOne({ baseUser: fakeId });
+  //     console.log(updatedNewContentCreator);
+  //     expect(updatedNewContentCreator.approved).toBe(true);
+  //   });
+  // });
+
+  // //Test Application Rejection
+  // describe('PUT /api/application/:id?reject', () => {
+  //   it('Should reject an application with a reason', async () => {
+  //     const fakeId = newUser._id;  // Assuming newUser is already defined in your test setup
+  //     const reason = 'Not meeting the criteria';
+
+  //     await request(app)
+  //       .put(`/api/application/${fakeId}/reject`)  // Updated the route
+  //       .send({ reason })  // Sending the reason in the request body
+  //       .expect(200);  // Expect a successful rejection
+
+  //     const updatedNewContentCreator = await db.collection('content-creators').findOne({ baseUser: fakeId });
+  //     expect(updatedNewContentCreator.rejected).toBe(true);
+  //   });
+  // });
```

Comment:
Its fine to comment these out since they are broken and not because of you, just remember to notify whoever made them and tell them to fix it. On top of that you can also tell them that tests shouldnt query the database, instead they should mock the data needed.

Reply:
Det var det vi kommenterede ud sammen i tirsdag. Jeg mener, det var Lasse, der havde skrevet testene.

Reply:
Ikke mig

Reply:
From this commit: https://github.com/Educado-App/educado-backend/commit/ffaa651f45aa8d4fe7dc3699218c2c5df3256fe7#diff-5f576f2166487c5d54c2e7f9d7e1b26cee7657950b54d2799024fa10fbf5927c

Reply:
Yeah I remember them now, I simply modified them a little by mocking the sendmail function since the OG tests was taking too long to complete, it was gruppe 3 that made those tests for their PBI if I recall correctly, so you should probaly tell them to fix them if they are broken.

Reply:
I will resolve the conversation but keep this in mind to group 3 for the next Sprint